### PR TITLE
topology1: do not apply dynamic pipeline on RTNR tplg

### PR DESCRIPTION
**Pull request by johnylin76**
Enabling dynamic pipeline on RTNR-integrated topologies will causes DSP crashes.

By now dynamic pipeline is planned to be applied on all adl/rpl topologies without third-party modules. RTNR should be excluded as well.

### CODE REVIEW

**Comment by johnylin76** [comment]
@Vamshigopal PTAL

File: tools/topology/topology1/CMakeLists.txt [hunk 1]
```diff
@@ -153,17 +153,17 @@ set(TPLGS
...
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682-pdm1\;-DCODEC=MAX98357A\;-DFMT=s16le\;-DDMIC_DAI_LINK_16k_PDM=STEREO_PDM1\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682-pdm1-google-aec\;-DCODEC=MAX98357A\;-DFMT=s16le\;-DDMIC_DAI_LINK_16k_PDM=STEREO_PDM1\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DCHANNELS=2\;-DGOOGLE_RTC_AUDIO_PROCESSING"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682-pdm1-drceq\;-DCODEC=MAX98357A\;-DFMT=s16le\;-DDMIC_DAI_LINK_16k_PDM=STEREO_PDM1\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DDRC_EQ"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-rt1011-rt5682\;-DCODEC=RT1011\;-DFMT=s24le\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1"
-	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682-rtnr\;-DCODEC=MAX98357A\;-DFMT=s16le\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DCHANNELS=2\;-DRTNR"
+	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682-rtnr\;-DCODEC=MAX98357A\;-DFMT=s16le\;-DPLATFORM=tgl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DCHANNELS=2"
```
Comment (Vamshigopal) [inline tools/topology/topology1/CMakeLists.txt:162]: You have removed RTNR here , if it updated by mistake please correct it.

**Review by kv2019i — APPROVED** [review]
@johnylin76 Could you send this to stable-v2.2 as well (this is now the upstream for these tplg files). Otherwise looks good.

**Review by Vamshigopal — APPROVED** [review]

**Comment by marc-hb** [comment]
@singalsu is there some testbench quick fix that could be cherry-picked in this branch to fix this `parse_topology()` mem corruption?

https://github.com/thesofproject/sof/actions/runs/7420846006/job/20192979309?pr=8699

If complicated forget it.

**Comment by singalsu** [comment]
> @singalsu is there some testbench quick fix that could be cherry-picked in this branch to fix this `parse_topology()` mem corruption?
> 
> https://github.com/thesofproject/sof/actions/runs/7420846006/job/20192979309?pr=8699
> 
> If complicated forget it.

Thanks, I don't remember but I need to check this.

**Comment by johnylin76** [comment]
> @johnylin76 Could you send this to stable-v2.2 as well (this is now the upstream for these tplg files). Otherwise looks good.

Sure. Thanks for the note.
The cherry-pick is https://github.com/thesofproject/sof/pull/8705

**Review by btian1 — APPROVED** [review]
just one comments, could you change 3p --> third party in your commit message?

**Comment by johnylin76** [comment]
> just one comments, could you change 3p --> third party in your commit message?

Changed :)

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@mwasko fyi - topology change that improves stability.

**Pull request merged**